.idea/navEditor.xml

```diff
@@ -394,131 +394,13 @@
             <LayoutPositions>
               <option name="myPositions">
                 <map>
-                  <entry key="activityFragment">
-                    <value>
-                      <LayoutPositions>
```

Comment:
![KreyGasm](https://user-images.githubusercontent.com/54945977/95749517-fa13d500-0c9b-11eb-984e-3ebe20f38615.png)
I approve the deletions